src/pool/srv_pool.c

```diff
@@ -62,6 +62,8 @@ struct pool_svc {
 	struct ds_pool	       *ps_pool;
 };
 
+static bool pool_disable_evict = false;
```

Comment:
I'd suggest we disable auto exclusion by default.

Reply:
Hmm, yes, I thought about that. but this is per node, instead of per pool.  I can not find something similar for node as pool svc step_up.  Any suggestion?

Reply:
Not yet. Without a command to enable and disable auto exclusion, landing this PR may cause troubles. Our SWIM timeouts are pretty short. And there's no way to add an excluded server back.

Reply:
hmm, sorry a bit confuse here. I was told DAOS needs to respond to node DEAD event automatically.   You mean we can not trust SWIM? or I misunderstand sth?

Reply:
SWIM, with proper parameters, should be accurate most of the time. I guess you can try landing this and see if it might break anybody's workflow...